x/stakeibc/keeper/host_zone.go

```diff
@@ -189,6 +189,36 @@ func (k Keeper) IncrementValidatorSlashQueryProgress(
 	return nil
 }
 
+// Increments the number of validator delegation changes in progress by 1
+// Note: This modifies the original host zone struct. The calling function must Set this host zone
+// for changes to persist
+func (k Keeper) IncrementValidatorDelegationChangesInProgress(hostZone *types.HostZone, validatorAddress string) error {
```

Comment:
If we compute the rebalance ICAs all at once but then send them in small batches of 5 messages, would we want to add 5 to the count or just one and then when we get the call back we would also decrement 1.   This counter is per validator, so some batches of 5 messages in rebalance ICAs would be several to the same validator like [A->B. A->C, A->D, B->C, B->E] so would this be A.changes+=3 B.changes+=2?  Should we update the validators who are receiving since their delegation is changing even if they didn't get the message so it would be A.changes+=3 B.changes+=3 C.changes+=2 D.changes+=1 E.changes +=1

Reply:
I think it's easiest to just increment both source and destination validator for each redelegation message